vscode/language-configuration.json

```diff
@@ -27,5 +27,10 @@
         ["(", ")"],
         ["\"", "\""],
         ["'", "'"]
-    ]
+    ],
+    "wordPattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s\\$]+)",
```

Comment:
This line comes from here: https://code.visualstudio.com/api/language-extensions/language-configuration-guide

I just removed the `\\-` and added a `\\$`.